bundles/org.openhab.binding.ventaair/src/main/java/org/openhab/binding/ventaair/internal/message/CommandMessage.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.ventaair.internal.message;
+
+import org.openhab.binding.ventaair.internal.message.action.Action;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Message containing a command to be send to the device
+ *
+ * @author Stefan Triller - Initial contribution
+ *
+ */
+public class CommandMessage extends Message {
```

Comment:
You could move DTOs in a package called `dto` to get rid of the checkstyle warnings about missing null annotations.